src/Command/DebugMappingCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DebugMappingCommand extends Command
```

Comment:
I would call it `DebugCommand` (that is consistent with the test and the command name)